src/Microsoft.PowerShell.Commands.Management/commands/management/GetComputerInfoCommand.cs

```diff
@@ -5152,17 +5130,6 @@ private static class PInvokeDllNames
         [return: MarshalAs(UnmanagedType.Bool)]
         public static extern bool GetFirmwareType(out FirmwareType firmwareType);
 
-        /// <summary>
-        /// Convert a Local Identifier to a Locale name.
-        /// </summary>
-        /// <param name="localeID">The Locale ID (LCID) to be converted.</param>
-        /// <param name="localeName">Destination of the Locale name.</param>
-        /// <param name="localeNameSize">Capacity of <paramref name="localeName"/></param>
-        /// <param name="flags"></param>
-        /// <returns></returns>
-        [DllImport(PInvokeDllNames.LCIDToLocaleNameDllName, SetLastError = true, CharSet = CharSet.Unicode)]
-        public static extern int LCIDToLocaleName(uint localeID, System.Text.StringBuilder localeName, int localeNameSize, int flags);
```

Comment:
Please also remove the definition of PInvokeDllNames.LCIDToLocaleNameDllName at line 5115.

Reply:
Fixed